### PR TITLE
feat(insights): module empty states

### PR DESCRIPTION
**Pull request by DominikB2014**
Everything here is internal only for now, this PR updates the module onboarding, and wraps the other modules with the onboarding component. Any old onboarding content will be removed in the release PR.

Text that should be used is found here https://www.notion.so/sentry/Insights-Pending-for-M-Launch-4c9162936fbc4a6897085851c503c0f1

Also note: there is duplicate images between this PR and the module upsells (https://github.com/getsentry/getsentry/pull/14519/files), I didn't realize they would share the same images. 
Once this is merged, i will delete the `getsentry` duplicates and use the ones from `sentry`

image

### CODE REVIEW

File: static/app/views/insights/cache/settings.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import type {SpanMetricsQueryFilters} from 'sentry/views/insights/types';
 
 export const MODULE_TITLE = t('Caches');
 export const BASE_URL = 'caches';
+export const DATA_TYPE = t('Cache');
+export const DATA_TYPE_PLURAL = MODULE_TITLE;
```
Comment (gggritso) [inline static/app/views/insights/cache/settings.ts:7]:
```suggestion
export const DATA_TYPE_PLURAL = t('Caches');
```

Safer, so they can be changed independently. If you change the `MODULE_TITLE` to `t('Cache Insights')` or something, it'd look weird in other places

File: static/app/views/insights/queues/settings.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import {t} from 'sentry/locale';
 
 export const MODULE_TITLE = t('Queues');
 export const BASE_URL = 'queues';
+export const DATA_TYPE = t('Queue');
```
Comment (gggritso) [inline static/app/views/insights/queues/settings.ts:5]: 🤔 not `t('Task')`?

**Review by gggritso — COMMENTED** [review]
Looks good! Watch out for the plurals though, they shouldn't reference the module title!

File: static/app/views/insights/cache/settings.ts (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/insights/cache/settings.ts:7]: Yeah that's a good point, I'll change it! 

I was trying to decide between reusing module title or not, but your of point of it being safer to not reuse it makes sense

File: static/app/views/insights/queues/settings.ts (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/insights/queues/settings.ts:5]: Spoke with Ed, We're going to keep it as Queue

**Review by gggritso — APPROVED** [review]
👍🏻

**Pull request merged**